src/index.js

```diff
@@ -163,28 +169,35 @@ OAuth.prototype.refresh = function (url, callback) {
  *
  * @param  {String}   url      The token endpoint URL
  * @param  {String}   clientId The client id
+ * @param  {String}   scopes The scopes with oauth
  * @param  {Function} callback First arg is error (if any), second is OAuth object on success.
  */
-OAuth.authorizeImplicitly = function(url, clientId, callback) {
-	var self = this,
+OAuth.authorizeImplicitly = function (url, clientId, scopes, callback) {
+	var scope = scopes || '',
 		win,
 		webview,
-		retryCount = 0,
-		state = generateGUID();
+		state = generateGUID(),
+		webViewDetails;
+
 	win = Ti.UI.createWindow(AUTH_WINDOW_OPTIONS);
-	webview = Ti.UI.createWebView({
-		width : '100%',
-		height : '100%',
-		url : buildURL(url, {
-			//scope: 'scopes', // FIXME Allow user to specify scopes?
+	webViewDetails = {
+		width: '100%',
+		height: '100%',
+		url: buildURL(url, {
+			scope: scope,
 			approval_prompt: 'force',
 			redirect_uri: CALLBACK_URL,
 			response_type: 'token',
 			client_id: clientId,
 			btmpl: 'mobile',
 			state: state
 		})
-	});
+	};
+	if (OAuth.ignoreSslError.get('value')) {
```

Comment:
I just merged PR #1 which has a simpler way to set this value, ditch your re-implementation of this flag and rebase on master with Hans' changes for this flag.